runelite-client/src/main/java/net/runelite/client/plugins/virtuallevels/VirtualLevelsPlugin.java

```diff
@@ -93,19 +115,22 @@ public void onScriptCallbackEvent(ScriptCallbackEvent e)
 				intStack[intStackSize - 1] = Experience.MAX_VIRT_LEVEL;
 				break;
 			case "skillTabTotalLevel":
-				int level = 0;
-
-				for (Skill s : Skill.values())
+				if (config.alterTotalLevel())
 				{
-					if (s == Skill.OVERALL)
+					int level = 0;
+
+					for (Skill s : Skill.values())
 					{
-						continue;
-					}
+						if (s == Skill.OVERALL)
+						{
+							continue;
+						}
 
-					level += Experience.getLevelForXp(client.getSkillExperience(s));
-				}
+						level += Experience.getLevelForXp(client.getSkillExperience(s));
 
-				stringStack[stringStackSize - 1] = TOTAL_LEVEL_TEXT_PREFIX + level;
+					}
```

Comment:
This should be moved to the line above it to maintain the empty line between the loop and setting of the stringStack.